src/libs/actions/connections/index.ts

```diff
@@ -238,9 +240,8 @@ function updateManyPolicyConnectionConfigs<TConnectionName extends ConnectionNam
 function hasSynchronizationError(policy: OnyxEntry<Policy>, connectionName: PolicyConnectionName, isSyncInProgress: boolean): boolean {
     // NetSuite does not use the conventional lastSync object, so we need to check for lastErrorSyncDate
     if (connectionName === CONST.POLICY.CONNECTIONS.NAME.NETSUITE) {
-        return !isSyncInProgress && !!policy?.connections?.[CONST.POLICY.CONNECTIONS.NAME.NETSUITE].lastErrorSyncDate;
+        return !isSyncInProgress && policy?.connections?.[connectionName]?.verified === false;
```

Comment:
I think we shouldn't use verified. We should use `lastErrorSyncDate` as earlier.